packages/react/package.json

```diff
@@ -27,14 +27,15 @@
   },
   "dependencies": {
     "@aws-amplify/ui": "3.0.1-next.0",
+    "@radix-ui/react-tabs": "0.0.16",
     "@xstate/react": "^1.4.0",
     "autoprefixer": "^10.3.1",
     "classnames": "^2.3.1",
     "deepmerge": "^4.2.2",
     "nanoid": "^3.1.25",
     "postcss-js": "^3.0.3",
-    "qrcode": "^1.4.4",
-    "@aws-amplify/ui-react-v1": "npm:@aws-amplify/ui-react@1.2.9"
+    "@aws-amplify/ui-react-v1": "npm:@aws-amplify/ui-react@1.2.9",
```

Comment:
My original PR somehow omitted this package. Eric C told me that it needs to be included.